### PR TITLE
Flyswatter changes.

### PR DESCRIPTION
**Pull request by Thelolswat**
BUFF! FLYSWATTER! WHAT MORE COULD YOU WANT

:cl: TheLoLSwat
tweak: After seeing how extremely unbalanced the flyswatter was, we at Nanotrasen have decided to use an all new material for these, ultimately making stronger flyswatters
/:cl:

[why]: # Mostly testing out making PRs with this, bigger projects will come soon

### CODE REVIEW

**Comment by ghost** [comment]
Proud of you

**Review by JohnGinnane — DISMISSED** [review]
😅 💦

**Comment by JohnGinnane** [comment]
I was actually kinda hoping the flyswatter would do extra damage against moths

**Comment by Carbonhell** [comment]
Sure, +1
I dont really want to see this damaging moths more tho unless it is kept below force 10

**Comment by JohnGinnane** [comment]
Yeah just more as a meme thing, similar to the laser stunning cats

**Comment by GuyonBroadway** [comment]
The swatter already does bonus damage to moths. Not as much as it does to flypeople but it does.

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -516,7 +516,7 @@
 	icon = 'icons/obj/weapons.dmi'
 	icon_state = "flyswatter"
 	item_state = "flyswatter"
-	force = 1
```
Comment (GuyonBroadway) [inline code/game/objects/items/weapons/weaponry.dm]: Tut tut, not modular.

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]

**Comment by praisenarsie** [comment]
this is redundant anyway since barely anybody plays janitor

**Comment by ghost** [comment]
flyswatters are part of the loadout system, making this CHANGE OF THE YEAR

**Comment by praisenarsie** [comment]
i don't like this anyway since this will allow you to crit flies and moths in 5 hits (or more if they have a bit of armor)

File: code/game/objects/items/weapons/weaponry.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/weapons/weaponry.dm]: [Spank spank spank! Naughty programmer!](https://what.thedailywtf.com/topic/15001/enlightened)

**Comment by ghost** [comment]
+1

**Comment by bardofrage** [comment]
Congrats on your first PR. +1

File: code/game/objects/items/weapons/weaponry.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/weapons/weaponry.dm]: Amari don't you have fun features to be removing from other servers?

File: hippiestation/code/game/objects/items/weapons/weaponry.dm [hunk 2]
```diff
@@ -0,0 +1,546 @@
...
+/obj/item/weapon/melee/baseball_bat/ablative/IsReflect()//some day this will reflect thrown items instead of lasers
+	var/picksound = rand(1,2)
+	var/turf = get_turf(src)
+	if(picksound == 1)
+		playsound(turf, 'sound/weapons/effects/batreflect1.ogg', 50, 1)
+	if(picksound == 2)
+		playsound(turf, 'sound/weapons/effects/batreflect2.ogg', 50, 1)
+	return 1
+
+/obj/item/weapon/melee/flyswatter
```
Comment (GuyonBroadway) [inline hippiestation/code/game/objects/items/weapons/weaponry.dm]: all you need in this entire file is
``/obj/item/weapon/melee/flyswatter
	force = 2``

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/wizard/soulstone.dm [hunk 3]
```diff
@@ -113,7 +113,7 @@
 ////////////////////////////Proc for moving soul in and out off stone//////////////////////////////////////
 
 
-/obj/item/device/soulstone/proc/transfer_soul(choice as text, target, mob/user).
+/obj/item/device/soulstone/proc/transfer_soul(choice as text, target, mob/user)
```
Comment (GuyonBroadway) [inline code/game/gamemodes/wizard/soulstone.dm:116]: why did you commit this? change it back.

**Review by GuyonBroadway — DISMISSED** [review]

File: code/game/gamemodes/wizard/soulstone.dm (hunk 3 above)
Comment (Manly-Dwarf) [reply code/game/gamemodes/wizard/soulstone.dm:116]: Pretty sure he forgot to branch and instead just did it all on his Master.

Comment (tortellinitony) [inline code/game/gamemodes/wizard/soulstone.dm:116]: Get rid of this change.

File: hippiestation/code/game/objects/items/weapons/weaponry.dm [hunk 4]
```diff
@@ -0,0 +1,546 @@
...
+	name = "Flyswatter"
+	desc = "Useful for killing insects of all sizes."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "flyswatter"
+	item_state = "flyswatter"
+	force = 2
```
Comment (tortellinitony) [inline hippiestation/code/game/objects/items/weapons/weaponry.dm]: This and `/obj/item/weapon/melee/flyswatter` are the only lines you need to keep. Get rid of the rest of the lines.

**Review by tortellinitony — CHANGES_REQUESTED** [review]
Also, if this is a new file you have added, you need to implement changes in `hippiestation.dme`

File: hippiestation/code/game/objects/items/weapons/weaponry.dm [hunk 5]
```diff
@@ -1,546 +0,0 @@
-/obj/item/weapon
```
Comment (Manly-Dwarf) [inline hippiestation/code/game/objects/items/weapons/weaponry.dm]: yes I think I see the problem here

**Review by Manly-Dwarf — CHANGES_REQUESTED** [review]

**Comment by Thelolswat** [comment]
Kawaii-Big-Boss helped me through the issues i was having, closing and making a new and improved PR by tommorow

**Pull request closed without merging**